content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+
+There are three types of contributions you'll come across:
+
+- Adding contributions to declarations
+- Updating contributions to declarations
+- Removal contributions to declarations
+
+The method used to review each of these types of contribution varies, and you'll find a detailed explanation of this as you proceed with this document.
+
+### Where to find the contributions
+
+These contributions are housed in the [`contrib-declarations`](https://github.com/OpenTermsArchive/contrib-declarations) repository under the pull requests tab. You can find the pull requests [here](https://github.com/OpenTermsArchive/contrib-declarations/pulls).
+
+## How To Review Pull Requests that Add Declarations
+
+When a contribution is made using the contribution tool, the OTA-bot will create a pull request. This pull request typically consists of several checks, which guides the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
+You should focus should be on two things: accuracy and quality.
+
+- Accuracy is about making sure that the contribution document is accurate and tracks **significant** sections that actually make a legal impact when changed.
+- Quality is about making sure that the contribution is of high quality and uses stable CSS selectors that will make sure the document will need less maintainace over time.
+
+### Step-by-step Review Guide
+
+1. Click on the inspect the declaration suggestion link to view contribution using the contribution tool.
+2. Use the link provided in the URL section of the contribution tool to check out the original document.
+3. Verify that the name of the service matches the JSON file and complies with the [guidelines](https://docs.opentermsarchive.org/guidelines/declaring/#service-name).
+4. Quickly scan the document to ensure that the correct term type has been selected. To determine the term type, consider who the intended audience is and what the document is discussing. You can also refer to the [terms type guide](https://github.com/OpenTermsArchive/terms-types/blob/main/termsTypes.json) to find the best term type for the document.
+5. Confirm that the selected area of the document contains only one term type and does not include any other types.
+6. Check both the significant and insignificant parts of the document.
+    - Ensure that only [simple/basic selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors) are used and avoid complex selectors like [combinators](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#combinators) and [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp). To simplify, try to avoid selectors with indexes/numbers e.g `:nth-child(n)` and long chains e.g. `div > body > .aside > p > span > a )`
```

Comment:
This is very good guidance! Unfortunately, sometimes it is necessary to use complex selectors when the document is very complex.

```suggestion
    - Ensure that, as much as possible, only [simple/basic selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors) are used and avoid complex selectors like [combinators](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#combinators) and [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp). To simplify, try to avoid selectors with indexes/numbers e.g `:nth-child(n)` and long chains e.g. `div > body > .aside > p > span > a )`
```

Reply:
Ok. I haven't faced situations where it's necessary to use complex selectors. But that is a great perspective to share. I'll take your change into account